_posts/en/newsletters/2022-03-23-newsletter.md

```diff
@@ -0,0 +1,115 @@
+---
+title: 'Bitcoin Optech Newsletter #192'
+permalink: /en/newsletters/2022/03/23/
+name: 2022-03-23-newsletter
+slug: 2022-03-23-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter summarizes discussion about the speedy trial soft
+fork activation mechanism and links to an update of an optimized LN
+pathfinding algorithm.  Also included are our regular sections with
+descriptions of recent changes to services and client software,
+announcements of new releases and release candidates, and summaries of
+notable changes to popular Bitcoin infrastructure software.
+
+## News
+
+- **Speedy trial discussion:** a mention of the speedy trial [soft fork
+  activation method][topic soft fork activation] from the summary of a
+  recent meeting about the proposed [OP_CHECKTEMPLATEVERIFY][topic
+  op_checktemplateverify] opcode was spun off into a separate thread
+  on the Bitcoin-Dev mailing list for additional discussion after Jorge
+  Timón [expressed][timon st] concern about the use of speedy trial for
+  a soft fork he thought was controversial.
+
+    Russell O'Connor [explained][oconnor st] how the concerns had been
+    previously addressed.  Anthony Towns further [described][towns st]
+    how an unwanted soft fork activation using speedy trial could be
+    resisted by objecting users.
+
+- **Pathfinding algorithm update:** René Pickhardt [posted][pickhardt
+  pathfinding] to the Lightning-Dev mailing list that he'd found a much
+  more computationally effecient approximation for his and Stefan
+  Richter's pathfinding algorithm published last year.  See [Newsletter
```

Comment:
Actually Carsten Otto has just merged a first version at https://github.com/C-Otto/lnd-manageJ/issues/6 and tweeted about it: https://twitter.com/c_otto83/status/1506017622747488259 but I am not sure if it makes more sense to mention that once it is more stable (I haven't even had the chance to review it properly yet). So far I think it can not conduct the payments as lnd does not provide a gRPC interface for MPP payments.